lib/appoptics_apm/inst/httpclient.rb

```diff
@@ -144,6 +144,7 @@ def do_get_stream_with_appoptics(req, proxy, conn)
           raise e
         ensure
           # AppOpticsAPM::API.log_exit('httpclient', kvs.merge('Async' => 1))
+          kvs[:Backtrace] = AppOpticsAPM::API.backtrace if AppOpticsAPM::Config[:httpclient][:collect_backtraces]
```

Comment:
Why is this in the `ensure` block as opposed to up at the beginning of the block?

Reply:
Right!
I guess I was easing into a pattern, need to move it up.